pdh_json_updater/update_json.py

```diff
@@ -100,7 +100,8 @@ def main():
     for code in codes_to_update:
         update_json_with_set(code, existing_json)
 
-    FileHandler.save_format_json_to_file(existing_json, new_last_set_release_date)
+    FileHandler.save_format_json_to_file(
```

Comment:
I think my linter settings are off...

Reply:
Hahaha what the...

Reply:
Im surprised that is legal python syntax

Reply:
Would expect it to be more like 
```
FileHandler.save_format_json_to_file(
    existing_json, new_last_set_release_date
)
```